content/docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers.md

```diff
@@ -27,10 +43,20 @@ Required job dependencies cause the job to fail if the required product asset is
 Jobs can generate multiple product assets, but a job might not need all product assets from another job that was declared as a job dependency. In this scenario, the required product assets are specified for the job dependency. If the hash of any of the specified product assets changes, the job with the dependency is run.
 
 A material, for example, has a job dependency on a shader. The shader job generates multiple product assets including assets that contain shader logic and a shader configuration. The material job dependency only needs the configuration from the shader, not the logic. The material job runs when the shader configuration is changed, not when the shader logic is changed.
+
+### Declaring job dependencies
+
+One of the fields on the JobDescriptor is m_jobDependencyList, a vector of JobDependency objects. The Job Dependency is populated with the same information as a SourceFileDependency to identify what file to track, as well as information to identify the associated job to use as a dependency, the type of job dependency, and the optional list of product sub IDs.
+
+Like with source dependencies, the source file for a job dependency can be declared in a few different ways. UUIDs are the most stable way to track these dependencies.
+
+If a path must be used to the source asset, it can be an absolute path or relative, and can make use of wildcards. If a path needs to be used instead of a UUID, absolute paths can be more stable than relative paths here because this data is specific to the local machine, so pathing differences across machines won't matter. Relative paths cannot include directory change markers across root drives, so if a source asset is in a scan folder on one drive root (such as C:/Projects/MyGame/Assets) and a source dependency is in a folder on another drive root (such as E:/Gems/SomeGem/Assets), a relative path will not be usable.
+
+Source assets, used as the target of a job dependency, must be in scan directories. Read more about [scan directories here.](/docs/user-guide/assets/pipeline/scan-directories/)
```

Comment:
As above, adding code tags per style guide.

```suggestion
One of the fields on the `JobDescriptor` is `m_jobDependencyList`, a vector of `JobDependency` objects. The Job Dependency is populated with the same information as a `SourceFileDependency` to identify what file to track, as well as information to identify the associated job to use as a dependency, the type of job dependency, and the optional list of product sub IDs.

Like with source dependencies, the source file for a job dependency can be declared in a few different ways. UUIDs are the most stable way to track these dependencies.

If a path to the source asset must be used, it can be an absolute path or relative, and can make use of wildcards. If a path needs to be used instead of a UUID, absolute paths can be more stable than relative paths here because this data is specific to the local machine, so pathing differences across machines won't matter. Relative paths cannot include directory change markers across root drives, so if a source asset is in a scan folder on one drive root (such as `C:/Projects/MyGame/Assets`) and a source dependency is in a folder on another drive root (such as `E:/Gems/SomeGem/Assets`), a relative path will not be usable.

Source assets, used as the target of a job dependency, must be in scan directories. Read more about [scan directories here.](/docs/user-guide/assets/pipeline/scan-directories/)
```